packages/kuma-gui/src/app/external-services/views/ExternalServiceDetailView.vue

```diff
@@ -21,31 +21,38 @@
           })"
           v-slot="{ data }"
         >
-          <KCard
+          {{ console.log('test', data) }}
```

Comment:
👀  just so you notice this, interesting it wasn't caught by a our linter tho 🤔

Reply:
Oh wow, nice catch! I'll check why it wasn't catched 😄

Reply:
Removed and opened https://github.com/kumahq/kuma-gui/pull/3311